site/content/entry/yuzu-progress-report-aug-2020/index.md

```diff
@@ -0,0 +1,209 @@
+  >}}
+
+After taking a look at this first concept, they decided that it wasn't very intuitive for the user. One morning, as Morph was about to finish the backend implementation, [Rei](https://github.com/Its-Rei) had an epiphany to replicate the Switch's native UI for the Controller Applet and whipped up the concept design you see below:
+
+{{< imgs
+    "./secondapplet.png| The one that started it all"
+  >}}
+
+As you can see, this was much closer to the design we have now, but was lacking some features such as the number of supported controllers, led pattern, border color, and explain text.
+Wanting to get as close to the Switch's design as possible, Morph implemented these features one by one, and the frontend UI was adapted by Rei to include these features, bringing us to the final design we have now.
+
+{{< single-title-imgs
+    ""
+    "./supportedplayers.png"
+    "./ledpatterns.png"
+  >}}
+
+{{< single-title-imgs
+    "It's all coming together now"
+    "./bordercolor.png"
+    "./explaintext.png"
+  >}}
+
+As the design was now complete, Morph hooked up all of these features from the backend to the frontend and produced these clean and pretty results!
+
+Below are comparisons between the Nintendo Switch's and yuzu's Controller Applets:
+
+{{< single-title-imgs
+    "Mario Kart 8 Deluxe"
+    "./mariokart8deluxe-switch.png"
+    "./mariokart8deluxe-yuzu.png"
+  >}}
+
+{{< single-title-imgs
+    "Kirby Star Allies"
+    "./kirbystarallies-switch.png"
+    "./kirbystarallies-yuzu.png"
+  >}}
+
+{{< single-title-imgs
+    "Super Mario Maker 2"
+    "./smm2-switch.png"
+    "./smm2-yuzu.png"
+  >}}
+
+### There is still more to come!
+
+There is always room for improvement, keep your eyes peeled for future additions and features!
+
+As we cannot fit in all the changes present in both parts, for our more tech savvy users, you can find all the changes for them through the following links: [[Mjölnir Part 1]](https://github.com/yuzu-emu/yuzu/pull/4530) [[Mjölnir Part 2]](https://github.com/yuzu-emu/yuzu/pull/4597)
+
+## Bugfixes and improvements
+
+[Khronos](https://www.khronos.org/) recently added the new `VK_EXT_extended_dynamic_state` extension to the Vulkan API, and driver support is slowly being completed by the vendors. As to be expected with fresh support for this extension, you can expect stuff to be in a *non-perfect* state, or as we like to say, "Expect Dragons". AMD recently added support for this extension in the 20.8.3 Windows driver but part of it is glitched, resulting in graphical errors in games like `Super Mario Odyssey`. [Rodrigo](https://github.com/ReinUsesLisp) temporarily solved this by [blacklisting the extension on the AMD Windows drivers until it is fixed](https://github.com/yuzu-emu/yuzu/pull/4599). Intel doesn't support `VK_EXT_extended_dynamic_state` at the time of writing this, and Nvidia had already fixed their issues with the extension.
```

Comment:
Change `As to be expected with fresh support for this extension, you can expect stuff to be in a non-perfect state, or as we like to say, “Expect Dragons”.`

to

`Per the norm with newly supported extensions, you can expect stuff to be in a *non-perfect* state, or as we like to say, "Expect Dragons."`